app/views/users/show.html.slim

```diff
@@ -42,10 +42,10 @@ nav.actions
     h4 About me
     p = render_markdown(@user.bio).html_safe
 
-  - if @user.teams.any?
+  - if @user.visible_teams.any?
```

Comment:
I'm still not 100% sure if I got the feature and the issue right. The way I understand it, teams set to "invisible" should be hidden from the profile until they are accepted. That's also how I read the current implementation of the `visible` method on the `team`:
```ruby
def visible
  where("teams.invisible IS NOT TRUE OR teams.kind IN (?)", KINDS)
end
```
So once a team is accepted, it's always visible. Which also means, the chaining of scopes you do in the user model should actually not even be necessary. It should be enough to do a
```ruby
@user.teams.visible.any?
```